### PR TITLE
[bitnami/contour] add ipFamilies to contour envoy service.yaml

### PR DESCRIPTION
**Pull request by Uburro**
### Description of the change

my task is to create a service with the load balancer type ipv6 only, for this you need to specify ipFamilies

### Benefits

IPv6 only service in dualstak custer
### Applicable issues

- fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/contour/values.yaml [hunk 1]
```diff
@@ -840,6 +840,10 @@ envoy:
     ## @param envoy.service.ipFamilyPolicy [string], support SingleStack, PreferDualStack and RequireDualStack
     ##
     ipFamilyPolicy: ""
+    ## @param envoy.service.ipFamilies [array] List of IP families (e.g. IPv4, IPv6) assigned to the service.
+    ## Ref: https://kubernetes.io/docs/concepts/services-networking/dual-stack/
+    ipFamilies: []
+    ##  - IPv6
```
Comment (andresbono) [inline bitnami/contour/values.yaml:846]: Just a minor comment, we tend to set the examples above the parameters. Does it make sense for you as well?

```suggestion
    ## E.g.
    ## ipFamilies:
    ##   - IPv6
    ##
    ipFamilies: []
```

**Review by andresbono — COMMENTED** [review]
Thank you very much for your contribution! Could you please take a look at my suggestion?

File: bitnami/contour/values.yaml (hunk 1 above)
Comment (Uburro) [reply bitnami/contour/values.yaml]: hi, fixed

**Review by andresbono — APPROVED** [review]
LGTM. Thank you for addressing the suggestion!

**Pull request merged**